library/src/main/java/com/airbnb/android/airmapview/MapboxWebViewMapFragment.java

```diff
@@ -22,4 +22,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
     return view;
   }
+
+  @Override public void setMapType(MapType type) {
+    // TODO: Not implemented
```

Comment:
@felipecsl You're referring to different maps like "Road, Satellite, Hybrid" correct?  If so, to get that to work for Mapbox a MapId with that style is needed.